public_html/wp-content/themes/pattern-directory/page-my-patterns.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * The template for displaying My Patterns.
+ *
+ * @package WordPressdotorg\Pattern_Directory\Theme
+ */
+
+namespace WordPressdotorg\Pattern_Directory\Theme;
+
+get_header();
+?>
+
+<main id="main" class="site-main" role="main">
+
+	<header class="entry-header">
+		<?php the_title( '<h1 class="entry-title">', '</h1>' ); ?>
+		<a href="<?php echo esc_url( home_url( '/new-pattern' ) ); ?>" class="button button-outline"><?php esc_html_e( 'Create a new pattern', 'wporg-patterns' ); ?></a>
```

Comment:
This button doesn't have a focus state. It's hard to tell it's focused on chrome.